cmd/minishift/cmd/image/export.go

```diff
@@ -69,7 +69,7 @@ func exportImage(cmd *cobra.Command, args []string) {
 	}
 	err = handler.ExportImages(imageCacheConfig)
 	if err != nil {
-		atexit.ExitWithMessage(1, fmt.Sprintf("Export of container images failed: %v", err))
+		atexit.ExitWithMessage(1, fmt.Sprintf("Failed to export the container image: %v", err))
```

Comment:
so is it all images? usually I see usage of one (specified) thing or all things.